bin/roave-no-leaks.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\NoLeaks\CLI;
+
+use PHPUnit\TextUI\Command;
+
+(function () {
+    require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
Using this package as a composer dependency so this path won't work. Had to change this line to `require_once __DIR__ . '/../../../../vendor/autoload.php';` to work. But maybe something like this makes more sense: https://github.com/php-api-clients/resource-generator/blob/master/api-client-resource-generator#L11

Reply:
Hmm, doesn't composer symlink it?

Reply:
Oh, bah, indeed, my mistake.